sympy/polys/constructor.py

```diff
@@ -57,9 +57,9 @@ def _construct_simple(coeffs, opt):
                 # this is a composite domain, e.g. ZZ[X], EX
                 return None
 
-    # Use the maximum precision of all coefficients for the RR or CC
+    # Use the minimum precision of all coefficients for the RR or CC
     # precision
-    max_prec = max(c._prec for c in float_numbers) if float_numbers else 53
+    max_prec = min(c._prec for c in float_numbers) if float_numbers else 53
```

Comment:
Rename it to `min_prec`?

Otherwise, this change seems to very much make sense!

Reply:
myopia! I made a change here and in the composite section